src/components/MaterialUI/DataDisplay/Table.js

```diff
@@ -384,10 +384,41 @@ const FullTableWithSavedScrollbar = React.forwardRef((props, ref) => {
 
 	useEffect(() => {
 		if (scrollbarViewState.scrollBarPosition > 0) {
+			// The Table component usually display the provided list ASAP however it can happen that the Table is not fully rendered with its items when we attempt to change the scrollTop value.
+			// The timer should ensure that the list is rendered before attempting to change the scrollTop if the first attempt was not able to set the scroll position
+			// 		This edge case was found using these steps:
+			//          * Create an organization with 50+ customers
+			//          * Go to the customer section of the organization and scroll down a bit
+			//          * Wait 1 second for the scroll position to be stored
+			//          * Go to the organization list (click on the tab)
+			//          * Go back to the organization (its tab should still be visible)
+			//          * We expect the customer list to remember its scroll position but it wasn't the case
+			//
+			// In addition to the timer, the tableName was added as a dependency to force the effect to execute again when it changes.
+			// This edge case was found using these steps:
```

Comment:
Is it normal that both edge cases have the same repro steps? If yes, is it really necessary to repeat it?